spacetemplate/assets/legacy.yaml

```diff
@@ -41,7 +46,10 @@ work_item_types:
   extends: "86af5178-9b41-469b-9096-57e5155c3f31"
   name: Experience
   can_construct: yes
-  description: TBD
+  description: An experience is a way to address a scenario, fundamental, or a
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.

Reply:
Fixed